docs/package.json

```diff
@@ -39,7 +39,7 @@
 		"start": "hugo server"
 	},
 	"dependencies": {
-		"@fluid-tools/api-markdown-documenter": "^0.8.3",
+		"@fluid-tools/api-markdown-documenter": "^0.9.0",
```

Comment:
What's the reasoning to bump this package in this PR?

Reply:
The new version is needed in order to specify heading names to `createThrowsSection` and `createExamplesSection`.

Reply:
Probably worth calling this out in the PR description.